Linux.md

```diff
@@ -2,36 +2,80 @@
+- **Lightweight.**
+    Linux's requirements for running on a system are much lower than that of Windows or Mac. With the right Linux distribution a user can have a modest setup and Linux will give the system value. Disk space and memory footprint can also be lower too. Some distributions are suitable for CPUs dating back to the Pentium family, others have a requirement of as little as 128MB of RAM and around the same amount for disk space!
+- **Heavily documented for beginners and advanced users alike.**
+    Linux has an active community of those willing to share their knowledge and help (much like FreeCodeCamp!). There are command line utilities built into Linux that provide documentation on commands, libraries, standards, etc. ([Man pages](https://en.wikipedia.org/wiki/Man_page) and [Info pages][2], and there is documentation available on the Internet in a variety of formats ([The Linux Documentation Project](http://www.tldp.org/), [LinuxQuestions](http://www.linuxquestions.org/), [ServerFault](http://serverfault.com/). There are self-paced certifications that can be taken that are recognized in the IT industry ([CompTIA's Linux+](https://certification.comptia.org/certifications/linux) and [LPI's LPIC tests](https://www.lpi.org)). Although not required, learning [C][3] gives the ability for a user to review the Linux code to see what they are running.
+- **Business support.**
+  The world depends on Linux supporting critical systems so the demand is not going away. This is important not only to those that help contribute to Linux, but also to those who support it (Jobs!). Linux support becomes more critical in IT, but also having that knowledge as a developer will make them more rounded and useful (i.e. [Full stack developers](https://medium.com/chris-messina/the-full-stack-employee-ed0db089f0a1#.ubttrv255)).
+- **Interoperability with other Operating Systems.**
+  Linux has driver support for NTFS and HFS+ filesystems (used by Windows and Macs) and also [Samba](https://www.samba.org/) for file/print service support on Windows machines. 
+- **Better hardware support.**
+  For other OSs, usually a user would have to go to the manufacturer's website to get driver support for different types of hardware. The Linux kernel supports a majority of hardware automatically via plug-and-play (largely in part because of the open source community).
+- **Software Availability/Compatibility.**
+    Linux has a rich set of applications that are available. It has [office applications](https://www.libreoffice.org/discover/libreoffice/), [browsers](https://www.google.com/chrome/browser/desktop/), [media players](http://www.videolan.org/vlc/), [image](https://www.gimp.org/)/[video processing](http://www.openshot.org/), etc. For video gaming, [Steam](http://store.steampowered.com/about/) runs on Linux which has many supported games. Windows applications can also run within Linux (see [Wine](https://www.winehq.org/) for details).
```

Comment:
in browsers provide Chrome **&** Mozilla. would be disrespectful not to mention Mozilla in a Linux about me :stuck_out_tongue: